sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -1561,6 +1562,63 @@ abstract class ParquetFilterSuite extends QueryTest with ParquetTest with Shared
       }
     }
   }
+
+  test("filter pushdown - local date") {
```

Comment:
can we just update `test("filter pushdown - date")` to test with DATETIME_JAVA8API_ENABLED on and off, so that we have less duplicated code?

Reply:
I removed it by the request https://github.com/apache/spark/pull/28272#discussion_r411529369 because the function is used in 2 places - filters and toJson. In filters, I wanna show we set it to false explicitly

Reply:
done